### ISSUE TITLE
Running tests gives error related to async(...)

### ISSUE THREAD

**Issue by rufuspollock** [datopian/data-cli#51]
Running tests gives error related to `async(...)`. Is this a babel issue or ...?

I'm running node v6.9.4.

```
/Users/rgrp/src/frictionlessdata/datahub-cli/lib/get.js:12
module.exports.get = async(pkgid) => {
                          ^
SyntaxError: Unexpected token (
    at Object.exports.runInThisContext (vm.js:76:16)
    at Module._compile (module.js:542:28)
    at Module._extensions..js (module.js:579:10)
    at extensions.(anonymous function) (/Users/rgrp/src/frictionlessdata/datahub-cli/node_modules/require-precompiled/index.js:16:3)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/rgrp/src/frictionlessdata/datahub-cli/node_modules/ava/lib/process-adapter.js:100:4)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/rgrp/src/frictionlessdata/datahub-cli/test/test_get.js:3:13)
    at Module._compile (module.js:570:32)
    at extensions.(anonymous function) (/Users/rgrp/src/frictionlessdata/datahub-cli/node_modules/require-precompiled/index.js:13:11)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/rgrp/src/frictionlessdata/datahub-cli/node_modules/ava/lib/process-adapter.js:100:4)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/rgrp/src/frictionlessdata/datahub-cli/node_modules/ava/lib/test-worker.js:49:1)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:509:3
```

**Comment by zelima** [comment]
according to [this article](https://www.infoq.com/news/2017/02/node-76-async-await) node officially released async/await support in 7.6.0 ... In the earlier versions it's considered to use `--harmony` flag at the beginning of the file `#!/usr/bin/env node --harmony`. 
but this is another problem as Linux kernel does not support multiple arguments in shebangs and fails with `No such file or directory" `

**Comment by anuveyatsu** [comment]
~I think we just need to check `node` version at runtime and include `babel-polyfill` if it is `<7.6.0`.~

We need to include babel to the project - add presets like es2015, stage-0. Then we need to add polyfill as well because just compiling es6 to es5 is not enough for async functions. Also need to think about tests - `ava` docs says that we can inherit from `.babelrc` file.

**Comment by anuveyatsu** [comment]
FIXED in #56 
We now support async functions and ES6 features in Node.js version 6.
